style.css

```diff
@@ -0,0 +1,220 @@
+header {
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  color: white;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  position: relative;
+  top: 74px;
+  left: 230px;
+  border-radius: 4px;
+  width: 982px;
+  height: 35px;
+}
+
+/* Selector for the grid container which contains my navbar Items */
+.flex-container {
+  text-align: center;
+  display: flex;
+  grid-template-columns: repeat(9, 1fr);
+}
+
+/* Selector for my grid items. Options inside my navbar */
+.flex-item {
+  flex: 1;
+  padding-top: 8px;
+  border-right: 1px solid #888;
+  vertical-align: middle;
+  height: 28px;
+}
+
+/* Selector for the apple logo fron font awesome */
+.fa-apple {
+  margin-top: -4px;
+  font-size: 22px;
+}
+
+/* Hover efect for all my grid items */
+.flex-item:hover {
+  cursor: pointer;
+  background: rgba(0, 0, 0, 0.3);
+}
+
+/* selector for the input field inside my navbar */
+.flex-item-search input {
+  padding: 0 10px;
+  color: black;
+  width: 100px;
+  border-radius: 8px;
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  border: 1px solid #888;
+  margin: 0 8px;
+  top: 5px;
+  position: relative;
+}
+
+/* Selector for the search icon from font awesome */
+.fa-search {
+  z-index: 100;
+  color: white;
+  position: absolute;
+  top: 8px;
+  left: 855px;
+}
+
+/* Selector for my main container */
+.container {
+  position: relative;
+  background: url('images/main_image.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: bottom;
+  margin: 0 auto;
+  width: 1442px;
+  height: 620px;
+  background-color: #eee;
+}
+
+/* Selector for the secondary options bellow my main container */
+.flex-other-topics {
+  width: 1442px;
+  margin: 3px auto 0 auto;
+  height: 200px;
+  display: flex;
+  grid-template-columns: repeat(4, 1fr);
+  grid-column-gap: 2px;
+}
+
```

Comment:
[HINT] The horizontal scrolling bar problem seems to be somewhere around these two css sections....
_/* Selector for my main container */_  and _/* Selector for the secondary options bellow my main container */_

Reply:
I think I have finished now hahaha